pom.xml

```diff
@@ -30,7 +30,7 @@
 		<logback.version>1.1.11</logback.version>
 		<selenium.version>3.0.0</selenium.version>
 		<build.server.version>1.2.0</build.server.version>
-		<git.server.version>1.0.11-SNAPSHOT</git.server.version>
+		<git.server.version>1.0.11</git.server.version>

```

Comment:
Integration Tests seem to pass, there are two unit / integration tests that need to be fixed:

```
BuildResultsTest.before:46 » NullPointer
PullRequestsTest.testCreatePullRequest:52 » NullPointer
```

Probably some missing stubbing / data for fields that you introduced.